posthog/api/cohort.py

```diff
@@ -240,22 +238,31 @@ def update(self, cohort: Cohort, validated_data: dict, *args: Any, **kwargs: Any
 
         is_deletion_change = deleted_state is not None and cohort.deleted != deleted_state
         if is_deletion_change:
+            relevant_team_ids = (
+                Team.objects.order_by("id").filter(project_id=cohort.team.project_id).values_list("id", flat=True)
```

Comment:
curious why we need `order_by("id")` here

Reply:
Good question, I don't remember 😄 It's definitely superfluous here, removed